docs/src/modules/utils/generateMarkdown.js

```diff
@@ -345,13 +345,13 @@ function generateClasses(reactAPI) {
 
 ${text}
 
-You can override the style of the component thanks to one of these customizability points:
+You can override the style of the component thanks to one of these customization points:
 
 - With a rule name of the [\`classes\` object prop](/customization/components/#overriding-styles-with-classes).
 - With a [global class name](/customization/components/#overriding-styles-with-global-class-names).
 - With a theme and an [\`overrides\` property](/customization/globals/#css).
 
-If it's not enough, you can find the [implementation of the component](${SOURCE_CODE_ROOT_URL}${normalizePath(
+If that's not sufficient, you can check the [implementation of the component](${SOURCE_CODE_ROOT_URL}${normalizePath(
     reactAPI.filename,
   )}) for more detail.
```

Comment:
```suggestion
   )}) for more details.
```

Reply:
It seems to depend if we want to focus on the quality of the quantity of extra information: https://english.stackexchange.com/questions/123672/detail-or-details-read-on-for-more-detail-details?

Reply:
Note, we were using "detail" in the past, highlighting the quality aspect. We are also using "detail" when the ref is attached to an imperative handle.